packages/fuels/tests/logs/script_logs/src/main.sw

```diff
@@ -2,6 +2,15 @@ script;
 
 use std::logging::log;
 
+impl AbiEncode for (TestStruct, TestEnum) {
```

Comment:
I assume this will be auto implemented by the compiler at some point. Can we we have a PR to track the removal of these?

Reply:
https://github.com/FuelLabs/fuels-rs/issues/1260